src/app.jsx

```diff
@@ -207,18 +234,25 @@ const NavigatorCardBody = ({ currentFilter, files, isGrid, setPath, path, pathIn
     }
 };
 
-export const ViewSelector = ({ isGrid, setIsGrid }) => {
+export const ViewSelector = ({ isGrid, setIsGrid, sortBy, setSortBy }) => {
     const [isOpen, setIsOpen] = useState(false);
     const onToggleClick = isOpen => setIsOpen(!isOpen);
-    const onSelect = () => setIsOpen(false);
+    const onSelect = (ev, itemId) => {
+        setIsOpen(false);
+        setSortBy(itemId);
+    };
 
     return (
-        <Dropdown
+        <Select
+            id='sort-selector'
```

Comment:
Let's pick some names for the identifiers that are more close to what PF elements these are.

I suggest sort-menu instead of sort-selector.

Adn sort-menu-toggle instead of sorting-dropdown.